app_check/src/common/common.h

```diff
@@ -19,11 +19,18 @@ namespace firebase {
 namespace app_check {
 namespace internal {
 
+// Used by the function registry, so that other products can call App Check
```

Comment:
Is this comment accurate? I think this is the enum used by the Future API to identify the future, not the enum used by function registry.

Reply:
Whoops, you are right. Fixed